File: SourceAFIS/Matching/Minutia/IRootSelector.cs
using System;
using System.Collections.Generic;
using System.Text;
using SourceAFIS.Extraction.Templates;

namespace SourceAFIS.Matching.Minutia
{
    public interface IRootSelector
    {
        IEnumerable<MinutiaPair> GetRoots(Template probe, Template candidate);
    }
}

File: java/sourceafis/src/sourceafis/matching/minutia/IRootSelector.java
package sourceafis.matching.minutia;

import java.util.Enumeration;

import sourceafis.extraction.templates.Template;
 
public interface IRootSelector{
       //IEnumerable<MinutiaPair> GetRoots(Template probe, Template candidate);
	   Enumeration<MinutiaPair> GetRoots(Template probe, Template candidate);
}